bionty/__init__.py

```diff
@@ -68,10 +68,18 @@
 
    Read the guides:
 
-   - `Access public ontologies <https://lamin.ai/docs/public-ontologies>`__
+   - :doc:`docs:public-ontologies`
    - :doc:`docs:bio-registries`
 
-   For more background on how public ontologies are accessed, see `bionty.base`.
+Bionty base
+===========
+
+Bionty also offers a standalone version `bionty-base` that does not require an active lamindb instance.
```

Comment:
I'd say: The read-only {mod}`bionty.base` module doesn't require a lamindb instance, and enables reading from public ontologies.